k8s/namespaces/fees-pay/ccpay-bubble-frontend/prod.yaml

```diff
@@ -2,27 +2,13 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: ccpay-bubble-frontend
-  namespace: fees-pay
 spec:
   releaseName: ccpay-bubble-frontend
-  rollback:
-    enable: true
-    retry: true
-  chart:
-    git: git@github.com:hmcts/hmcts-charts
-    ref: master
-    path: stable/ccpay-bubble-frontend
   values:
     nodejs:
-      replicas: 2
-      useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ccpay/bubble-frontend:prod-687ee3f-20220221170611 #{"$imagepolicy": "flux-system:ccpay-bubble-frontend"}
```

Comment:
`image:` already referenced in ccpay-builb-frontend.yaml

Reply:
removed it from prod.yaml